obspy/clients/eida/client.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+FDSN Web Service client using EIDA routing and authentication.
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+    Helmholtz-Zentrum Potsdam - Deutsches GeoForschungsZentrum GFZ
+    (geofon@gfz-potsdam.de)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+from future.utils import PY2
+
+import io
+import obspy.clients.fdsn
+from . import fetch
+
+if PY2:
+    import urlparse
+
+else:
+    import urllib.parse as urlparse
+
+
+class Client(obspy.clients.fdsn.Client):
+    """
+    FDSN Web Service client using EIDA routing and authentication.
+
+    For details see the :meth:`~obspy.clients.eida.client.Client.__init__()`
+    method.
+    """
+    def __init__(self, base_url='GFZ', retry_count=10, retry_wait=60,
```

Comment:
I think the `retry_count` should default to `1` and the `retry_wait` to maybe at most 5 seconds. Right now it (worst case) hangs for 10 minutes. IMHO it is reasonable to assume that the web services are stable and in production. What is the reasoning behind this choice?